.changeset/eighty-squids-protect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-css': patch
+'@itwin/itwinui-react': patch
```

Comment:
When we change something in only `itwinui-css`, but that indirectly affects `itwinui-react`, do we include `itwinui-react` in the changeset also?

Reply:
yes, read https://github.com/iTwin/iTwinUI/wiki/Styling#v2